draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -490,17 +490,16 @@ RESET\_STREAM frame.
 QUIC allows an application to abort reading the stream and specify an error code
 {{Section 3.5 of !RFC9000}}. An RTP receiver can signal this to the RTP sender
 by sending a QUIC STOP\_SENDING frame, which is exported to the RTP sender
-through the ROQ API, as described in {{api-considerations}}. The RTP-sender
-SHOULD not interpret the reception of STOP\_SENDING as an indication that the
-RTP receiver lost interest in the media stream as a whole, but rather that a
-part of the media stream was not received timely. In this case, the late data
-could also delay the following media frames due to head-of-line blocking, which
-the sender can avoid by transmitting new media frames on new QUIC streams. A
-sender that receives STOP\_SENDING MUST NOT retransmit any media frames already
-sent on the QUIC stream, on which it received the STOP\_SENDING, on a new QUIC
-stream. An RTP sender SHOULD instead continue to send media frames on new QUIC
-streams starting with the first frame that was not transmitted on the stream
-that received STOP\_SENDING.
+through the ROQ API, as described in {{api-considerations}}. STOP\_SENDING is
+not an indication that the RTP receiver lost interest in the media stream as a
+whole, but rather that a part of the media stream was not received timely. In
+this case, the late data could also delay the following media frames due to
+head-of-line blocking, which the sender can avoid by transmitting new media
+frames on new QUIC streams. A sender that receives STOP\_SENDING MUST NOT
+retransmit any media frames already sent on the QUIC stream, on which it
+received the STOP\_SENDING, on a new QUIC stream. An RTP sender MUST continue to
+send media frames on new QUIC streams starting with the first frame that was not
+transmitted on the stream that received STOP\_SENDING.
 
```

Comment:
Are we mandating that RTP senders open a new stream and continue sending? 

If we are, and a RTP receiver really has stopped caring about this RTP stream, how does it "turn the RTP sender off"? 

I think we're basically telling a RTP receiver that wants to "skip forward to usable media frames", to transmit a "STOP\_SENDING" frame, triggering a "STREAM_RESET", and then to carry out another SDP exchange so the RTP sender **knows** that it should continue by sending the newest media frames on a new QUIC stream.

Do you see another way to ensure that the Right Thing happens?

Does that make sense?

Reply:
> Are we mandating that RTP senders open a new stream and continue sending?

Yes, I think that is what we should do.

> If we are, and a RTP receiver really has stopped caring about this RTP stream, how does it "turn the RTP sender off"?

I don't think using STOP_SENDING is the right tool for this. Instead, it would probably require some signaling.

> I think we're basically telling a RTP receiver that wants to "skip forward to usable media frames", to transmit a "STOP_SENDING" frame, triggering a "STREAM_RESET", and then to carry out another SDP exchange so the RTP sender **knows** that it should continue by sending the newest media frames on a new QUIC stream.
>
> Do you see another way to ensure that the Right Thing happens?

I don't understand why we would carry out another SDP exchange here. The sender should take STOP_SENDING as a signal that the receiver is still interested in the media stream in general, but wants to start over with new media frames on new QUIC streams.